PlaceRouteHierFlow/PnRDB/ReadConstraint.cpp

```diff
@@ -1,437 +1,471 @@
-#include "PnRdatabase.h"
-
-#include <iostream>
 #include <fstream>
 #include <iomanip>
+#include <iostream>
+
+#include "PnRdatabase.h"
 
 using namespace nlohmann;
 void PnRdatabase::ReadConstraint_Json(PnRDB::hierNode& node, const string& jsonStr) {
-    auto logger = spdlog::default_logger()->clone("PnRDB.PnRdatabase.ReadConstraint_Json");
-    json jedb = json::parse(jsonStr);
-    json constraints = jedb["constraints"];
-    for (auto constraint : constraints) {
-      if (constraint["const_name"] == "SymmNet") {
-        PnRDB::net tmpnet;
-        tmpnet.name = constraint["net1"]["name"];
-        for (auto pin : constraint["net1"]["blocks"]) {
-          if (pin["type"] == "pin") {  // if block pin
-            for (int i = 0; i < (int)node.Blocks.size(); i++) {
-              if (node.Blocks.at(i).instance.back().name == pin["name"]) {
-                for (int j = 0; j < (int)node.Blocks.at(i).instance.back().blockPins.size(); j++) {
-                  if (node.Blocks.at(i).instance.back().blockPins.at(j).name == pin["pin"]) {
-                    PnRDB::connectNode newnode = {PnRDB::Block, j, i};
-                    tmpnet.connected.push_back(newnode);
-                    break;
-                  }
+  auto logger = spdlog::default_logger()->clone("PnRDB.PnRdatabase.ReadConstraint_Json");
+  json jedb = json::parse(jsonStr);
+  json constraints = jedb["constraints"];
+  for (auto constraint : constraints) {
+    if (constraint["const_name"] == "SymmNet") {
+      PnRDB::net tmpnet;
+      tmpnet.name = constraint["net1"]["name"];
+      for (auto pin : constraint["net1"]["blocks"]) {
+        if (pin["type"] == "pin") {  // if block pin
+          for (int i = 0; i < (int)node.Blocks.size(); i++) {
+            if (node.Blocks.at(i).instance.back().name == pin["name"]) {
+              for (int j = 0; j < (int)node.Blocks.at(i).instance.back().blockPins.size(); j++) {
+                if (node.Blocks.at(i).instance.back().blockPins.at(j).name == pin["pin"]) {
+                  PnRDB::connectNode newnode = {PnRDB::Block, j, i};
+                  tmpnet.connected.push_back(newnode);
+                  break;
```

Comment:
Was the indentation broken in this file? A number of changes seem to be due to indentation only (?).

Reply:
This change has already been committed to master in a different PR. I don't know why it is being displayed here. (I did mess up a rebase at the beginning of the PR.)
If you want to see the real change wrt master, do a git diff master on the command line.